.github/workflows/build_SRWA.yml

```diff
@@ -0,0 +1,119 @@
+# This workflow Builds the UFS SRWEATHER APP
+name: Build SRWA
+
+on:
+# Enable the ability to manually run this workflow, as well as when the ci-aws-intel-build label is attached to a PR, and if the PR is updated
+  workflow_dispatch:
+  pull_request:
+    branches: 
+      - rrfs_ci 
+    types: 
+      - labeled
+
+# Use a default login shell which loads intel/oneapi/setvars.sh, and lmod path 
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  
+  Set_Repo_and_Branch:
+    name: Set the SRWA repo and branch names  
+    runs-on: self-hosted 
+    if: ${{ github.event.label.name == 'ci-aws-intel-build' }}
+    outputs: 
+      SRWA_REPO: ${{ steps.set_repo.outputs.SRWA_REPO }}
+      SRWA_BRANCH: ${{ steps.set_repo.outputs.SRWA_BRANCH }}
+      
+  # If a developer submits a concurrent PR to the App, set the SRWA_REPO & SRWA_BRANCH env variables to their repo and branch,
+  # if not, set the outputs to the rrfs_ci branch of NOAA-GSL/ufs-srweather-app  
+    steps:
+      - name: Set Repo and Branch Variables 
+        id: set_repo
+        run: |
+          new_branch=`git ls-remote https://github.com/${{ github.actor }}/ufs-srweather-app.git ${{ github.head_ref }}`
+          if [ -n "$new_branch" ]; then
+            echo "SRWA_REPO is being set to : ${{ github.actor }}/ufs-srweather-app.git" 
+            echo "::set-output name=SRWA_REPO::${{ github.actor }}/ufs-srweather-app"
+            echo "SRWA_BRANCH is being set to : " ${{ github.head_ref }}
+            echo "::set-output name=SRWA_BRANCH::${{ github.head_ref }}"
+          else
+            echo "There is not a head_ref with the same name as this PR in the SRW App, the default SRW App (NOAA-GSL fork) repository will be used."
+            echo "::set-output name=SRWA_REPO::NOAA-GSL/ufs-srweather-app"
+            echo "There is not a similar branch name associated with this PR in the SRW App, the default SRWA App branch (rrfs_ci) will be used."
+            echo "::set-output name=SRWA_BRANCH::rrfs_ci"
+          fi
+
+  # Clone the SRWA - with github.run_id in filepath to avoid multiple PR Conflicts, & track build tests by their run_id
+  # Relative paths are used here to decouple the running of this workflow from the self-hosted runner itself. We'd like to avoid the added complication of setting up the same directory structure for each SH runner.
+  # (Cloning repositories outside of the GITHUB_WORKSPACE directory is not supported with actions/checkout@v3, so it needs to be moved into the parent directory seperately)
+  # The SRWA is cloned within the GITHUB_WORKSPACE (default location of where 'run' commands are executed ), we need to move its location up one directory, as the regional_workflow is nested within the SRWA.
+  Clone_SRWA:
+    name: Clone SRWA and move into parent directory
```

Comment:
Why does it _need_ to move up one directory? Can you not clone SRW where you want it, and then clone regional_workflow inside that clone using the `path` option for both?

Reply:
@christinaholtNOAA, you pose some great questions, and this a much better approach. The GHA workflow now clones  SRW, and places the regional_workflow inside of that clone using the `path` option.